src/bin/tikv-ctl.rs

```diff
@@ -323,6 +324,33 @@ trait DebugExecutor {
         self.do_compact(db, cf, from, to);
     }
 
+    fn set_region_tombstone(&self, region: u64, endpoints: Vec<String>) {
+        let debugger = self.get_local_debugger().unwrap_or_else(|| {
+            eprintln!("This command is only for local mode");
+            process::exit(-1);
+        });
+        match RpcClient::new(&endpoints)
+            .unwrap_or_else(|e| perror_and_exit("RpcClient::new", e))
+            .get_region_by_id(region)
+            .wait()
+            .unwrap_or_else(|e| perror_and_exit("Get region id from PD", e))
+        {
+            Some(mut meta_region) => {
+                let epoch = meta_region.take_region_epoch();
+                let peers = meta_region.take_peers();
```

Comment:
Should check if peers contains target peer.

Reply:
This logic is in `Debugger::set_region_tombstone`, because here we can't get self `store_id`.

Reply:
we can get store id with `store_ident` key

Reply:
see https://github.com/pingcap/tikv/blob/master/src/raftstore/store/keys.rs#L37

Reply:
Yes, we do this in `Debugger::set_region_tombstone`.